server.js

```diff
@@ -2,32 +2,66 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+// Imported API
+import netflixData from './data/netflix-titles.json'
+
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Routes 
+
+//(Start page)
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('<h1>View these endpoints</h1> All netflixtitles => https://ebbabw-express-deploy.herokuapp.com/netflixtitles <br> Netflixtitles single id  =>  https://ebbabw-express-deploy.herokuapp.com/netflixtitles/80025678 <br> Filter on year and filter on movie and tv shows relesed on diffrent years => https://ebbabw-express-deploy.herokuapp.com/year/2019 and https://ebbabw-express-deploy.herokuapp.com/year/2019/?type=movie or https://ebbabw-express-deploy.herokuapp.com/year/2019/?type=tv%20show')
```

Comment:
You can create an HTML file and render it for the user here, or use a template engine like ejs: https://ejs.co/